src/stories/3-fields/select/select.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import React from "react";
+import styled from "styled-components";
+import { FrontendEngine, ISelectSchema, Select } from "../../..";
+import { ExcludeReactFormHookProps } from "../../common";
+
+const ExcludeDefaultProps = {
+	type: { table: { disable: true } },
+	id: { table: { disable: true } },
+	title: { table: { disable: true } },
+	options: { table: { disable: true } },
+};
+
+export default {
+	title: "Field/Select",
+	component: Select,
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Select</Title>
+					<Description>This component provides a set of options for use to select</Description>
+					<Heading>Props</Heading>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Examples" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...ExcludeReactFormHookProps,
+		...ExcludeDefaultProps,
+		schema: {
```

Comment:
- no need to explain the schema prop since devs are expected to integrate via JSON schema only
and each prop can do away with the `schema.` prefix
- missing `type` prop (it should have `SELECT` value`)
- `id` and `type` needs to be marked as required
- for each prop, can also define the control / type / defaultValue for easy tinkering via the control

can refer to textarea story too